discord/state.py

```diff
@@ -150,7 +148,7 @@ async def logging_coroutine(
     coroutine: Coroutine[Any, Any, T], *, info: str
 ) -> T | None:
     try:
-        await coroutine
+        return await coroutine
```

Comment:
Why have you added a return here?

Reply:
`-> T | None` if we have no return then it should be `-> None`, i can change to it if needed

Reply:
I think in the spirit of this PR, adjust the typehint instead (and looking through the usage of this function, the returned object is never even used)

Reply:
> I think in the spirit of this PR, adjust the typehint instead (and looking through the usage of this function, the returned object is never even used)

fine, changed in new commit!